tests/test_qvm.py

```diff
@@ -65,7 +65,7 @@ def test_identity_expectation(self, shots, qvm, compiler):
 
         dev._samples = dev.generate_samples()
 
-        res = np.array([dev.expval(O1), dev.expval(O2)])
+        res = np.array([dev.expval(O1.obs), dev.expval(O2.obs)])
```

Comment:
Was this somehow changed to not support having expvals called twice (thus the need to extract the identity here before calling `dev.expval`?

Reply:
Hmm, not sure I understand :thinking: The number of calls to `dev.expval` should not matter, we just need to extract the observable so that we're passing a `qml.operation.Observable` rather than a `MeasurementProcess`.

Reply:
Oh, I guess I'm simply wondering why it seemingly was working before (or maybe it wasn't) and why this change is needed. It seems like an obvious change.